.github/pull_request_template.md

```diff
@@ -8,6 +8,6 @@ https://p2pvalidator.atlassian.net/browse/
 
 ## Screenshots (Optional)
 
-Screenshot #1 | Screenshot #2 | Screenshot #3
-:------------:|:-------------:|:------------:
-[Paste Pic 1] | [Paste Pic 2] | [Paste Pic 3]
+| Screenshot #1 | Screenshot #2 | Screenshot #3 |
+|:-------------:|:-------------:|:-------------:|
+| [Paste Pic 1] | [Paste Pic 2] | [Paste Pic 3] |
```

Comment:
this may break the format and new lines, revert please or check it with 3 screenshots in a row

Reply:
oh yeah, sorry, surprise auto fix

Reply:
<img width="881" alt="Screen Shot 2023-02-10 at 16 54 09" src="https://user-images.githubusercontent.com/100791035/218047173-895385d6-8a7b-4167-8183-505d650b77c0.png">
it works
leave or revert?